avldb/build.sbt

```diff
@@ -5,7 +5,7 @@ name := "avldb"
 libraryDependencies ++= Seq(
   "javax.xml.bind" % "jaxb-api" % "2.4.0-b180830.0359",
   "ch.qos.logback" % "logback-classic" % "1.2.3",
-  "org.scorexfoundation" %% "scrypto" % "2.1.10"
+  "org.scorexfoundation" %% "scrypto" % "2.2.1"
```

Comment:
This change causes this in my local environment, but works on CI :thinking: 
```
[error] 285 errors were encountered during merge                                                                                                                                                                                                                                                                                                           
[error] java.lang.RuntimeException: deduplicate: different file contents found in the following:                                                                                                                                                                                                                                                           
[error] /home/lisak/.cache/coursier/v1/https/repo1.maven.org/maven2/org/bouncycastle/bcprov-jdk15on/1.64/bcprov-jdk15on-1.64.jar:org/bouncycastle/asn1/ASN1GeneralizedTime.class                                                                                                                                                                           
[error] /home/lisak/.cache/coursier/v1/https/repo1.maven.org/maven2/org/bouncycastle/bcprov-jdk15to18/1.66/bcprov-jdk15to18-1.66.jar:org/bouncycastle/asn1/ASN1GeneralizedTime.class                                                                                                                                                                       
```

Reply:
I guess we should exclude the `bcprov-jdk15to18` bouncy-castle dep

Reply:
Failed to get dependencyTree working, so fixed with

```
case PathList("org", "bouncycastle", xs @ _*) => MergeStrategy.first
```
for now